styles/_mixins.scss

```diff
@@ -33,15 +33,15 @@
       @content;
     }
   } @else if $point == medium-up {
-    @media (min-width: 768px) {
+    @media (min-width: 820px) {
       @content;
```

Comment:
on the subject of breakpoints, I've always wonder what this break point is specific to as this breakpoint is covered either by `medrium-only` or  `large`?

Reply:
The thinking with `medium-up` or `large` would be that if styles need to change between the two widths, so 820 and 1280px in our case, then you can use `medium-only` and then whatever other combination you need. If no styles need to change between those widths, you can just use `medium-up` for anything other than mobile.